src/com/company/Transport/dpo/Station.java

```diff
@@ -0,0 +1,32 @@
+package com.company.Transport.dpo;
+
+import com.company.Transport.dpo.mans.Passenger;
+
+import java.util.ArrayList;
+
+public class Station {
+    public ArrayList<Passenger> awaiting;
+    private String name;
+    private String timetable;
+
+    Station(String name, String timetable) {
```

Comment:
set access modifier as public - just default Station constructor is package-private